classes/services/queryBuilders/UserListQueryBuilder.inc.php

```diff
@@ -126,6 +144,21 @@ public function filterByRoleIds($roleIds) {
 		return $this;
 	}
 
+	/**
+	 * Set roles filter
+	 *
+	 * @param $userGroupIds int|boolean|array
```

Comment:
can it be boolean as well?

Reply:
No, good catch! I did some back-and-forth on how to best designate "show users with no roles". But shouldn't be boolean anymore.